pkg/apis/integreatly/v1alpha1/installation_types.go

```diff
@@ -68,8 +68,8 @@ var (
 	OperatorVersionAMQOnline             = "1.2.2"
 	OperatorVersionMonitoring            = "0.0.28"
 	OperatorVersionSolutionExplorer      = "0.0.32"
-	OperatorVersionRHSSO                 = "1.9.4"
-	OperatorVersionRHSSOUser             = "1.9.4"
+	OperatorVersionRHSSO                 = "1.9.5"
+	OperatorVersionRHSSOUser             = "1.9.5"
```

Comment:
Until Keycloak version is bumped in master

Reply:
this should be done now, if you rebase on master